packages/amplication-client/src/VersionControl/PendingChangeContent.tsx

```diff
@@ -18,7 +18,7 @@ const PendingChangeContent = ({
 }: Props) => {
   const { currentWorkspace, currentProject } = useContext(AppContext);
 
-  const url = `/${currentWorkspace?.id}/${currentProject?.id}/${change.resource.id}/${relativeUrl}`;
+  const url = `/${currentWorkspace?.id}/${currentProject?.id}/${change.origin.resource?.id}/${relativeUrl}`;
```

Comment:
by looking at the query in packages/amplication-client/src/Workspaces/queries/projectQueries.ts 
it looks like we fetch "change.resource" and not "change.origin.resource".

I assume there is a reason for this fix, but for my best understanding, it will not work - at least not in all cases 

<img width="519" alt="image" src="https://user-images.githubusercontent.com/43705455/189470700-9ecacfc5-a98b-467f-b62f-e963dfa8cc30.png">

Reply:
This fix is for the bug I told you about on the commits page, the link for the changes. In which cases do you think it will not work?

Reply:
I just don't understand how you get the value in origin.resource if we are not fetching this data from the server 

Based on the query, it looks like the original code is the correct one. I also saw that in action, so I am confused...

Reply:
@morhag90 can you please have a look at this

Reply:
@yuval-hazaz @abrl91 In our case, it will not work; as Yuval says, we don't get this value in the query, I checked this by printing the result, and it always returns undefined. 
@abrl91  In which case it breaks the URL??

Reply:
in Commits page => changes

Reply:
@morhag90 @yuval-hazaz undo my change. The purpose of this PR has been completed. I will open an issue about this bug.

Reply:
https://github.com/amplication/amplication/issues/3773